src/main/java/org/broadinstitute/hellbender/engine/AbstractConcordanceWalker.java

```diff
@@ -34,7 +33,7 @@
  * Created by Takuto Sato 1/30/17, abstractified by David Benjamin on 2/22/17.
  * {@link #onTraversalStart}, {@link #onTraversalSuccess} and/or {@link #closeTool}.
  */
-public abstract class AbstractConcordanceWalker extends GATKTool {
+public abstract class AbstractConcordanceWalker extends Walker {
```

Comment:
In the time since you opened this PR, more walker base classes have been added to the engine. Can you modify them to extend `Walker` as well?

Reply:
Fixed.